deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,146 @@ spec:
+    ## Start using the operator
 
-    ### Get the kubefedctl CLI tool
-
-    Before you use kubefed, you should ensure that you're using the binary
-    for this version. You can get it from the releases on GitHub.
+    This operator supports deploying a KubeFed control plane in both
+    namespace-scoped and cluster-scoped modes. For example, if you subscribe to
+    this package in a namespace `my-namespace`, KubeFed will be deployed to
+    manage objects _only_ in `my-namespace` in the host cluster and target
+    clusters.
+    
+    The operator provides the following CRDs for deploying components of a `KubeFed` control plane in a given cluster:
+    * KubeFedWebHook
+    * KubeFed
+    
+    ### KubeFed deployment
+    Follow these steps to deploy a KubeFed control plane:
+
+    * Install the operator by following the instructions in the
+    [documentation](https://docs.openshift.com/container-platform/4.1/applications/operators/olm-adding-operators-to-cluster.html#olm-installing-operators-from-operatorhub_olm-adding-operators-to-a-cluster).
+    The KubeFed operator only needs to be installed in the host cluster for the
+    deployment, not each individual target cluster.
+    ```
+    $ kubectl get po -n <namespace>
+      NAME                               READY   STATUS    RESTARTS   AGE
+      kubefed-operator-db58d9b89-8694x   1/1     Running   0          15s
+
+    ```
+    * Create a `KubeFedWebHook` resource to instantiate an admission webhook
+    controller for KubeFed. It's recommended to install this CR with
+    `scope:Cluster` to allow the webhook controller to serve API requests coming
+    from all KubeFed control planes within the same cluster.
+    ```
+    $ cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFedWebHook
+    metadata:
+      name: kubefed-webhook-resource
+    spec: 
+      scope: Cluster 
+    ---
+    ```
+    * Create a `KubeFed` resource to drive the installation of a KubeFed control
+    plane setting the scope to be either `Namespaced` or `Cluster`. If you're
+    planning to federate a cluster-scoped resource type like, for example
+    `StorageClass`, create this CR with `scope:Cluster`.
+    
+    ```
+    $ cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFed
+    metadata:
+      name: kubefed-resource
+    spec: 
+      scope: Cluster 
+    ---
+    ```
+    ```
+    $ kubectl get po -n <namespace> 
+    NAME                                          READY   STATUS    RESTARTS   AGE
+    kubefed-admission-webhook-77f7bd89dd-92kzj    1/1     Running   0          25s
+    kubefed-controller-manager-67d5d5cc99-5t2fk   1/1     Running   0          25s
+    kubefed-controller-manager-67d5d5cc99-b865z   1/1     Running   0          25s
+    kubefed-operator-db58d9b89-8694x              1/1     Running   0          61s
+    
+    ```
+    **Important Note**: 
```

Comment:
Move this note to earlier in the doc where you say we can deploy in either mode - keep information about this together ahead of the rest of the document so the user can make a decision earlier.